packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md

```diff
@@ -1,36 +1,36 @@
-### Your app displays the sign-in page
+### 1: Your app displays the sign-in page
 
-Build a sign-in page that captures both the user's name and their password.
+   Build a sign-in page that captures both the user's username and password.
```

Comment:
Their username may be something completely different than their name. So, we specify here in a conversational tone: user's name and their password.